### PR TITLE
Walindi pull request

### PR DESCRIPTION
**Pull request by walindi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -527,6 +527,32 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________*AlifSakib* Contributor card END ________  -->
 
+        <!-- ________ *walindi* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Walindi</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="_blank">@walindi_b</a>
```
Comment (TenTraicion) [inline index.html:535]:
```suggestion
            <a href="https://www.twitter.com/walindi_b" target="_blank">@walindi_b</a>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@walindi, Thanks for your contribution. kindly make the mentioned change and commit.

File: index.html (hunk 1 above)
Comment (walindi) [reply index.html]: Done

**Comment by CandidDeer** [comment]
Thank you for your contribution. I have merged your PR and your card will be live in a few moments.

**Pull request merged**